### PR TITLE
Create config.yml

### LINKED ISSUE #59: Create the config.yml file for the Welcome Bot
### Description
As a first time contributor,
I need a bot to help guide my first contribution,
so that I can follow this org's contribution guidelines.

### Mocks
image

### Acceptance Criteria

- [ ] newIssueWelcomeComment is complete with the correct links to this repo.

- [ ] newPRWelcomeComment is complete with the links to the project's stream or topic and the repo's Code of Conduct.

- [ ] firstPRMergeComment is complete.

### Additional context
great for first-timers

### PR DESCRIPTION
**Pull request by Amulya-coder**
### Description

Created config.yml file for the welcome bot to help first-time contributors. To activate the app you need to go to https://github.com/apps/welcome and click on install and give the app the required permissions.( Can only be done by a member of the org)

Fixes #59 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- Bugfix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by Amulya-coder** [comment]
@Aashipra I have try to tested on my local repository, by installing on my local repo and give the required permission but it's not working I'm unable to figure it out, can you please review it and tell me the reason here is the link to the repo:
https://github.com/Amulya-coder/Test-Bot

**Comment by Aaishpra** [comment]
No worries, let me check I will update you in some minutes

**Comment by Aaishpra** [comment]
@Amulya-coder can you test this in some other repo like another project where you have some files other than `.github` folder and `config.yml` actually I also worked on a similar issue a few months back (when I was eligible for a first-timer issue 😅  ) and faced similar problem while running it on a newly created empty repo and when I tested it on other repo(which had some other folders and files) it worked for me.
I don't specifically know the reason behind this, but I guess trying it, won't hurt. 😉

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+  
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉🎉🎉
```
Comment (Aaishpra) [inline .github/config.yml]: Another blank line here please (at the end of the file)

**Review by Aaishpra — CHANGES_REQUESTED** [review]
Just a minor tweak.

**Comment by Amulya-coder** [comment]
Ohkk @Aashipra I will try to test on some other repository without `.github` folder thanks for your suggestion will give you the updates asap.

File: .github/config.yml (hunk 1 above)
Comment (Amulya-coder) [reply .github/config.yml]: Done

File: .github/config.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats 🎉 for opening your very first issue in this project.AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md).🙌
+  You may submit a PR if you like! If you want to report a bug🐞 please follow our [Issue Template](https://github.com/anitab-org/.github/tree/main/.github/ISSUE_TEMPLATE). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
```
Comment (ashwinijha6) [inline .github/config.yml]: Just a few suggestions @Amulya-coder  there is no requirement of an exclamation `You may submit a PR if you like!` in this line

File: .github/config.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+  
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉🎉🎉
+
```
Comment (ashwinijha6) [inline .github/config.yml:35]: You can remove the extra line here

File: .github/config.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  Feel free to join us on [AnitaB.org Open Source Zulip Community](https://anitab-org.zulipchat.com/).💖 We have different streams for each active repository for discussions.✨ Hope you have a great time there!😄
+  
+
+# Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
+# Comment to be posted to on pull requests merged by a first time user
+
+
+firstPRMergeComment: >
+  Congrats on merging your first pull request! 🎉🎉🎉
```
Comment (ashwinijha6) [inline .github/config.yml]: Also you can add this line `We here at AnitaB.org are proud of you!` here refer [this](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/config.yml)

File: .github/config.yml (hunk 3 above)
Comment (Aaishpra) [reply .github/config.yml:35]: This extra line is needed, after the end of every file.

Comment (ashwinijha6) [reply .github/config.yml:35]: Is it necessary @Aaishpra to leave an extra line? I guess it is not compulsory

File: .github/config.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats 🎉 for opening your very first issue in this project.AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md).🙌
+  You may submit a PR if you like If you want to report a bug🐞 please follow our [Issue Template](https://github.com/anitab-org/.github/tree/main/.github/ISSUE_TEMPLATE). Also make sure you include steps to reproduce it and be patient while we get back to you.😄
```
Comment (ashwinijha6) [inline .github/config.yml]: I meant that removing an exclamation & adding a full stop. Could you pls do the necessary changes @Amulya-coder 
after `like`

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]

File: .github/config.yml (hunk 3 above)
Comment (Aaishpra) [reply .github/config.yml:35]: > Is it necessary @Aaishpra to leave an extra line? I guess it is not compulsory 

It's a good practice, read more about it [here](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline#:~:text=If%20content%20is%20added%20to,you%20actually%20wanted%20to%20see.)

File: .github/config.yml (hunk 5 above)
Comment (Amulya-coder) [reply .github/config.yml]: Done

File: .github/config.yml (hunk 3 above)
Comment (ashwinijha6) [reply .github/config.yml:35]: ok @Aaishpra I didn't know about that. Thankyou for the information 👍

**Comment by Amulya-coder** [comment]
@Aaishpra @ashwinijha6 I have made the suggested changes can you please review it.

**Review by Aaishpra — DISMISSED** [review]
LGTM @Amulya-coder

File: .github/config.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hello there!👋 Welcome to the project!💖
+  
+  
+  Thank you and congrats 🎉 for opening your very first issue in this project.AnitaB.org is an inclusive community, commited to creating a safe and positive environment.🌸 Please adhere to our [Code of Conduct](https://github.com/anitab-org/open-source-programs-web/blob/develop/CODE_OF_CONDUCT.md).🙌
```
Comment (ashwinijha6) [inline .github/config.yml]: @Amulya-coder  just noticed this could you please add a space after `project. AnitaB.org`

**Review by ashwinijha6 — CHANGES_REQUESTED** [review]

**Review by ashwinijha6 — APPROVED** [review]

**Review by Aaishpra — APPROVED** [review]

**Comment by ashwinijha6** [comment]
Thankyou for the contribution @Amulya-coder. Pinging @isabelcosta for the final review
@Aaishpra could you pls change the labels

**Pull request merged**